Demo/Sources/Recycling/GridViews/CompactMarkets/ToriCompactMarketsDemoView.swift

```diff
@@ -29,11 +29,11 @@ class CompactMarketsDemoView: UIView, Demoable {
 
 extension CompactMarketsDemoView: MarketsViewDataSource {
     func numberOfItems(inMarketsView marketsView: MarketsView) -> Int {
-        Market.newMarkets.count
```

Comment:
we don't want any changes to `CompactMarketsDemoView`

Reply:
👆

Reply:
changed the "Markets" to "torimarkets"

Reply:
but do we still want a market enum addition to the tori and finn markets?

Reply:
not sure what do you mean
we want to keep showing only finn markets here